libr/util/sys.c

```diff
@@ -923,69 +923,27 @@ R_API int r_is_heap (void *p) {
 
 R_API char *r_sys_pid_to_path(int pid) {
 #if __WINDOWS__
-	HANDLE kernel32 = GetModuleHandle (TEXT("kernel32"));
-	if (!kernel32) {
```

Comment:
all this deleted code is no longer necessary?

Reply:
It was faulty code and it never worked as intended.